.github/issue_template.md

```diff
@@ -8,6 +8,7 @@ Welcome! Before creating a new issue:
 * Look at the README *Troubleshooting* section
 * Search for relevant issues
 * Check that you have updated both the jupyterlab extension and the python package to the same version
```

Comment:
If adding a hint for how to check git version then there should probably also be a hint for these. Or similar to these command the hint for `git --version` should only appear in the context section